website/docs/d/advanced_cluster.html.markdown

```diff
@@ -55,13 +111,13 @@ In addition to all arguments above, the following attributes are exported:
 * `id` - The cluster ID.
 * `bi_connector_config` - Configuration settings applied to BI Connector for Atlas on this cluster. See [below](#bi_connector_config). **NOTE** Prior version of provider had parameter as `bi_connector`
 * `cluster_type` - Type of the cluster that you want to create.
-* `disk_size_gb` - Capacity, in gigabytes, of the host's root volume. **(DEPRECATED)** Use `replication_specs.#.region_config.#.(analytics_specs|electable_specs|read_only_specs).disk_size_gb` instead. Please refer [1.18.0 upgrade guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/1.18.0-upgrade-guide.html.markdown) for more details.
+* `disk_size_gb` - Capacity, in gigabytes, of the host's root volume. If `use_replication_spec_per_shard = true` then this value is equal to `disk_size_gb` of the first `replication_spec.#.region_configs.#.electable_specs`. **(DEPRECATED)** Use `replication_specs.#.region_config.#.(analytics_specs|electable_specs|read_only_specs).disk_size_gb` instead. Please refer [1.18.0 upgrade guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/1.18.0-upgrade-guide.html.markdown) for more details.
```

Comment:
If `use_replication_spec_per_shard = true` we still populate this root value, so believe this additional comment is not needed.

Reply:
This was added to address this point in [the ticket]([CLOUDP-260667](https://jira.mongodb.org/browse/CLOUDP-260667)) `Mention impact on root disk_size_gb computed value if new flag is used.` Is there anything we should mention on this? If not, I can remove the additional comment

Reply:
My bad for not updating that mention, after https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2405 logic was added to always populate at the root level to avoid breaking changes in the resource, and having that implementation in place it was added to the data source as well.

Reply:
discussed offline, removed